src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month month--length">
+      <div
+        class="month__day month__day--1 month__day--start-day--thu">
```

Comment:
```suggestion
      <div
        class="calendar calendar--28 calendar--thu"
      >
```
А тут уже набор классов, отвечающих за количество дней в месяце и отступ у первой ячейки